### PR TITLE
theme: add grandpa-style

### PR DESCRIPTION
**Pull request by TheTaylorLee**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description

Added a posh theme based off of the blue-owl theme. I modified some of the style choices for a different aesthetic.

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/grandpa-style.omp.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            "type": "prompt",
+            "alignment": "left",
+            "segments": [
+                {
+                    "type": "root",
+                    "style": "plain",
+                    "foreground": "#dd1e1e",
+                    "background": "#000000",
+                    "properties": {
+                        "prefix": " ",
```
Comment (JanDeDobbeleer) [inline themes/grandpa-style.omp.json]: that's the default value, can be removed here

File: themes/grandpa-style.omp.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                    "style": "plain",
+                    "foreground": "#ffffff",
+                    "background": "transparent",
+                    "properties": {
+                        "prefix": ""
+                    }
+                },
+                {
+                    "type": "path",
+                    "style": "diamond",
```
Comment (JanDeDobbeleer) [inline themes/grandpa-style.omp.json]: should be `plain`

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: themes/grandpa-style.omp.json (hunk 1 above)
Comment (TheTaylorLee) [reply themes/grandpa-style.omp.json]: Are you just talking about the background color or the prefix?

File: themes/grandpa-style.omp.json (hunk 2 above)
Comment (TheTaylorLee) [reply themes/grandpa-style.omp.json]: This has been changed.

File: themes/grandpa-style.omp.json (hunk 1 above)
Comment (TheTaylorLee) [reply themes/grandpa-style.omp.json]: I removed the prefix and believe that's what you were referring to. I'll leave this open for you to confirm. 

Thanks.

Comment (JanDeDobbeleer) [reply themes/grandpa-style.omp.json]: Yes, I was talking about the prefix. The comment should've been on that line.

**Comment by JanDeDobbeleer** [comment]
@all-contributors please add @TheTaylorLee and @PapiPeppers for design

**Pull request merged**